crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -34,7 +34,7 @@ mod assign;
 mod test;
 
 // NOTE: this must be kept in sync with the same constants in pbr.frag
-pub const MAX_UNIFORM_BUFFER_CLUSTERABLE_OBJECTS: usize = 256;
+pub const MAX_UNIFORM_BUFFER_CLUSTERABLE_OBJECTS: usize = 204;
```

Comment:
```suggestion
pub const MAX_UNIFORM_BUFFER_CLUSTERABLE_OBJECTS: usize = 204;
// A buffer of MAX_UNIFORM_BUFFER_CLUSTERABLE_OBJECTS GpuClusterableObject items must hold in 16384 bytes
const _: () =
    assert!(size_of::<GpuClusterableObject>() * MAX_UNIFORM_BUFFER_CLUSTERABLE_OBJECTS <= 16384);
```

let's check it at compile time